src/pages/room/[roomId].tsx

```diff
@@ -82,76 +82,82 @@ const Room: NextPage<RoomProps> = ({
+            <div className="rounded d-flex overflow-auto w-100">
+              <div className="w-100">
+                {room.threads.length > 0 ? (
+                  room.threads.map((thread) => (
+                    <div className="col mb-2" key={thread.id}>
+                      <div className="card text-bg-primary m-2 p-2 h-100 d-flex align-items-start">
+                        <div className="card-body w-100">
+                          <div className="row">
+                            <div className="col p-0">
+                              <Link
+                                key={thread.id}
+                                href={`/room/${room.id}/${thread.id}`}
+                              >
+                                <h2 className="card-title text-white text-decoration-underline">
+                                  {thread.name}
+                                  {" - "}
+                                  {thread.creator.name ?? "Unbekannt"}
+                                </h2>
+                                <p className="card-text text-secondary text-decoration-none">
+                                  {thread.description ??
+                                    "Dieser Thread hat keine genauere Beschreibung."}
+                                </p>
+                              </Link>
+                            </div>
+                            {isJoined && (
+                              <div className="col-auto p-0">
+                                <Button className="btn-secondary ms-auto">
+                                  Optionen {/*Popup für Optionen hinzufügen */}
+                                </Button>
+                              </div>
+                            )}
                           </div>
-                          {isJoined && (
-                            <Button className="btn-secondary ms-auto position-absolute top-0 end-0 p-2">
-                              Optionen {/*Popup für Optionen hinzufügen */}
-                            </Button>
-                          )}
                         </div>
                       </div>
-                    ))
-                  ) : (
-                    <div className="text-center">Es gibt keine Threads.</div>
-                  )}
-                </div>
+                    </div>
+                  ))
+                ) : (
+                  <div className="text-center">Es gibt keine Threads.</div>
+                )}
               </div>
             </div>
           </div>
           <div className="text-white p-2">
-            {/*Infos über den Room für die Rauminfos/Raumoptionen, müssen später an Raumoptionen-Componente übergebn werden und hier rausgelöscht werden*/}
+            <p>
+              Infos über den Room für die Rauminfos/Raumoptionen, müssen später
+              an Raumoptionen-Componente übergeben und hier rausgelöscht werden
+            </p>
```

Comment:
Gibt es einen Grund, warum das in dem Raum angezeigt werden soll?

Reply:
nein, danke für den Hinweiß.